spec/tests/ColumnLineageDatasetFacet/1.json

```diff
@@ -1,30 +1,173 @@
 {
   "columnLineage": {
     "fields": {
-      "order_day_of_week": {
+      "NAME": {
         "inputFields": [
           {
-            "field": "order_placed_on",
-            "name": "public.top_delivery_times",
-            "namespace": "postgres://postgres:5432"
+            "namespace": "SnowflakeOpenLineage",
+            "name": "CUSTOMERS",
+            "field": "NAME",
+            "transformations": [
+              {
+                "type": "DIRECT",
+                "subtype": "IDENTITY",
+                "masking": false
+              }
+            ]
+          },
+          {   
```

Comment:
there are some trailing whitespaces caught by our pre-commit `prettier` hook, if you remove them then everything will be good. 

For checking this kind of things locally I recommend installing [pre-commit](https://pre-commit.com/) and running `pre-commit install` in project directory. We have some hooks configured and before each commit `pre-commit` will run a check based on our configuration

Reply:
Thanks for this advice! Used pre-commit on the update to remove the deprecated transformationType, and it worked like a charm.